lib/html-min.cls.php

```diff
@@ -166,9 +170,27 @@ public function process()
 		return $this->_html;
 	}
 
+	/**
+	 * From LSCWP 6.2: Changed the function to test for special comments that will be skipped. See: https://github.com/litespeedtech/lscache_wp/pull/622
+	 */
 	protected function _commentCB($m)
 	{
-		return (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['))
+		// Check if is IE conditional comment
+		$is_ie_comment = (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['));
+
+		// Check if comment text is present in Page Optimization -> HTML Settings -> HTML Keep comments
+		$is_skip_comment = false;
+		if(count($this->_skipComments) > 0){
+			foreach ($this->_skipComments as $comment) {
+				if (!empty($comment) && strpos($m[1], $comment) !== false && $is_skip_comment === false) {
```

Comment:
1. Is there any special reason to use `===`?
2. Any reason to have `$is_skip_comment` in cond?
3. Why use `!empty($comment)` while not `$comment`?

Reply:
2) when I reviewed the code I added the break and forgot to remove the condition
1, 3) It's a safe writing for me. changed to your suggestion